pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -515,43 +514,43 @@ def _sanitize_request(
                 ]
         return payloads
 
-    # def _track_task_progress(
-    #         self,
-    #         worker_id : str,
-    #         remote_host: str,
-    #         remote_base_path: str,
-    #         remote_task_id = str,
-    #         # jwt: Optional[str] = None,
-    #         timeout : Optional[int] = None,
-    # ) -> None:
-    #     """Asynchronously track the task request on the remote TES.
-    #
-    #             Args:
-    #                 worker_id: Identifier for the background job.
-    #                 remote_host: Host at which the TES API is served that is
-    #                     processing this request; note that this should
-    #                     include the path information but *not* the base path
-    #                     path defined in the TES API specification; e.g.,
-    #                     specify https://my.tes.com/api if the actual API is
-    #                     hosted at https://my.tes.com/api/ga4gh/tes/v1.
-    #                 remote_base_path: Override the default path suffix
-    #                     defined in the TES API specification,
-    #                     i.e., `/ga4gh/tes/v1`.
-    #                 remote_task_id: Task identifier on remote WES service.
-    #                 jwt: Authorization bearer token to be passed on with
-    #                      task request to external engine.
-    #                 timeout: Timeout for the job. Set to `None` to disable
-    #                          timeout.
-    #             """
-        # task__track_run_progress.apply_async(
-        #     None,
-        #     {
-        #         'jwt': jwt,
-        #         'worker_id': worker_id,
-        #         'remote_host': remote_host,
-        #         'remote_base_path': remote_base_path,
-        #         'remote_task_id': remote_task_id,
-        #     },
-        #     soft_time_limit=timeout,
-        # )
-        # return None
+    def _track_progress(
```

Comment:
It kinda doesn't make sense to define a separate function here, as it's doing only a single call. Just call it directly. Might be that I have it in proWES like that, didn't check, if so, perhaps you could fix it here and then raise a PR in proWES as well. It's really unnecessarily bloating the codebase and repeating function documentation.

Reply:
Done